app/Http/Transformers/CampaignTransformer.php

```diff
@@ -18,10 +18,10 @@ public function transform(Campaign $campaign)
         return [
             'id' => $campaign->id,
             'internal_title' => $campaign->internal_title,
-            'start_date' => $campaign->start_date,
-            'end_date' => $campaign->end_date,
-            'created_at' => $campaign->created_at,
-            'updated_at' => $campaign->updated_at,
+            'start_date' => $campaign->start_date->toIso8601String(),
+            'end_date' => $campaign->end_date->toIso8601String(),
```

Comment:
Since `end_date` is optional, we should wrap this in [`optional()`](https://laravel.com/docs/5.7/helpers#method-optional).

```php
'end_date' => optional($campaign->end_date)->toIso8601String(),
```

Otherwise, we'll run into errors trying to call `->toIso8601String` on `null`.

Reply:
oh great call! Updated :)